README.md

```diff
@@ -44,8 +44,52 @@ You can find the client for the Fronts tool in [fronts-client](/fronts-client).
 
 - If you are developing locally and do not have frontend credentials from janus, the fronts tool won't have permissions to push events to the sqs queue that Facia-Press reads from. To test that a front is pressed, you will have to deploy your changes to code, and test the code from there.
 
-#### Troubleshooting
-##### Postgres
+
+## Different tools in this codebase
+
+### The Fronts Tool
+The most important part of this app is the Fronts Tool, used to curate the web and app front pages of the Guardian. Additionally it is used to manually curate emails.
+
+In the UI, users can move stories from the Content API feed into different front pages or emails. These fronts can then be launched or published.
+
+The Fronts Tool uses the most up-to-date front end, a React-Redux app located in [fronts-client](https://github.com/guardian/facia-tool/tree/master/fronts-client)
+
+Useful things to know:
+* An api call to Ophan pulls in page view data to allow users to understand at a glance that story's performance in the context of the front it is in
+
+* Containers can be shared between fronts. An update to a container in one place updates it everywhere
+
+* Certain containers have geolocation properties. This means they can only be seen by users in the location specified
+
+* Interactive atoms can be pasted into a front as CAPI links. This allows designers and the interactives team to create special content eg banners showing election results. Apart from articles, interactive atoms are the only other type of CAPI content allowed on Fronts
+
+* In the Email editor only - free text boxes can be created. This allows email editors to add introductions to their emails by hijacking the headline field. Rich text editing is enabled for this using Prosemirror.
+
+
+### The Editions creator
+
+The Editions interface is used to curate content on the Editions app (currently known as The Daily on iOS and Android). Editions also uses the fronts-client front end, and can be accessed from the Manage Editions menu on the [homepage](https://fronts.code.dev-gutools.co.uk/v2).
+
+Curation works in the same way as the main fronts tool. But there are these differences:
+
+* Fronts are part of Issues (normally one issue per day). You need to create or open a valid issue to make changes
+
+* Editions are published once instead of being continually updated and launched
+
```

Comment:
Can anyone @akash1810 @AWare @JustinPinner ? shed more light on the workings of the Editions app. 
-> Are there any notable user differences from main fronts
-> Do editions also go through fronts-press / dotcom etc or is the process different?